.eslintrc.yaml

```diff
@@ -28,3 +28,20 @@ rules:
   no-use-before-define: off
   object-curly-newline: off
   react/prop-types: off
+
+overrides:
+  -
+    files: ['src/**/*.js']
+    globals:
+      process: true
+      require: true
```

Comment:
Should we just set `env` to include `node` here (same as L46-47 below)?

Reply:
Well, they aren't actually in a node environment and only needed those two globals to pass linting, so I figured I'd be conservative. The webpack config files on L46-47 *are* node modules